### PR TITLE
Added face-recognition-resnet100-arface model from ONNX Zoo

### PR DESCRIPTION
**Pull request by Anna-Mironova**
- [x] Added GFLOPs and MParams

### CODE REVIEW

File: demos/face_recognition_demo/python/README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ The following pretrained models can be used:
 
 * `face-detection-retail-0004` and `face-detection-adas-0001`, to detect faces and predict their bounding boxes;
 * `landmarks-regression-retail-0009`, to predict face keypoints;
-* `face-reidentification-retail-0095` or `Sphereface`, to recognize persons.
+* `face-reidentification-retail-0095`, `Sphereface` or `face-recognition-resnet100-arcface-onnx`, to recognize persons.
```
Comment (eaidova) [inline demos/face_recognition_demo/python/README.md:14]: you forgot about https://github.com/openvinotoolkit/open_model_zoo/tree/master/models/public/facenet-20180408-102900

File: demos/smart_classroom_demo/cpp/README.md [hunk 2]
```diff
@@ -36,10 +36,7 @@ This file can be used as a parameter for [Model Downloader](../../../tools/downl
 ### Supported Models
 
 * face-detection-adas-0001
-* face-recognition-mobilefacenet-arcface
-* face-recognition-resnet100-arcface
-* face-recognition-resnet34-arcface
-* face-recognition-resnet50-arcface
+* face-recognition-resnet100-arcface-onnx
```
Comment (eaidova) [inline demos/smart_classroom_demo/cpp/README.md:39]: here is also not all acceptable models

File: demos/face_recognition_demo/python/README.md (hunk 1 above)
Comment (Anna-Mironova) [reply demos/face_recognition_demo/python/README.md]: Done

File: demos/smart_classroom_demo/cpp/README.md (hunk 2 above)
Comment (Anna-Mironova) [reply demos/smart_classroom_demo/cpp/README.md:39]: Done

File: demos/face_recognition_demo/python/README.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ The following pretrained models can be used:
 
 * `face-detection-retail-0004` and `face-detection-adas-0001`, to detect faces and predict their bounding boxes;
 * `landmarks-regression-retail-0009`, to predict face keypoints;
-* `face-reidentification-retail-0095` or `Sphereface`, to recognize persons.
+* `face-reidentification-retail-0095`, `Sphereface`, `facenet-20180408-102900` or `face-recognition-resnet100-arcface-onnx`, to recognize persons.
```
Comment (eaidova) [inline demos/face_recognition_demo/python/README.md:14]:
```suggestion
* `face-reidentification-retail-0095`, `Sphereface`, `facenet-20180408-102900` or `face-recognition-resnet100-arcface-onnx` to recognize persons.
```

**Review by eaidova — APPROVED** [review]

**Comment by vladimir-dudnik** [comment]
@Anna-Mironova please resolve merge conflict

File: demos/tests/cases.py [hunk 4]
```diff
@@ -526,6 +530,7 @@ def single_option_cases(key, *args):
         TestCase(options={'-m_lm': ModelArg('landmarks-regression-retail-0009')}),
         TestCase(options={'-m_reid': ModelArg('Sphereface')}),
         TestCase(options={'-m_reid': ModelArg('face-reidentification-retail-0095')}),
+        TestCase(options={'-m_reid': ModelArg('face-recognition-resnet100-arcface-onnx')}),
```
Comment (vladimir-dudnik) [inline demos/tests/cases.py:533]:
```suggestion
        TestCase(options={'-m_reid': ModelArg('face-recognition-resnet100-arcface-onnx')}),
        TestCase(options={'-m_reid': ModelArg('facenet-20180408-102900')}),
```

**Review by vladimir-dudnik — CHANGES_REQUESTED** [review]
add facenet to tests cases

File: demos/tests/cases.py (hunk 4 above)
Comment (Anna-Mironova) [reply demos/tests/cases.py:533]: Done

**Review by eizamaliev — APPROVED** [review]

**Pull request merged**